src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
That's true. I hadn't noticed that before. If that's what we want to do then we don't need the `ROS_PACKAGE_PATH_NO_DEFAULTS` option, because the `ROS_PACKAGE_PATH` will always be set by the `/opt/ros/kinetic/setup.*sh` files, and so when they are sourced the `/usr/share` will not be added.

All that being said, I still think behaving like `gcc` is less surprising, i.e. some include paths are always added unless you tell it not to include them automatically and adding your own include paths do not remove those default ones.